### DISCUSSION TITLE
Nx + Micro-Frontend with different technologies

### DISCUSSION THREAD

**Discussion by rambabupadimi** [nrwl/nx#15749] [General]
Is it possible to create architecture like  below using Nx
   1. Angular app (Eg: Shop) as a host 
   2. React App (Eg: Checkout) and Angular App (Eg: Cart) as remote apps  (microfrontends)

**Comment by meisterveda** [comment]
Would like to see this integration, would allow for teams to work in different frameworks, under module federation as a monorepo always was intended

**Comment by zackarydev** [comment]
Or between old versions of react and newer versions
